dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/NodeType.java

```diff
@@ -18,5 +18,5 @@
 package org.apache.dolphinscheduler.common.enums;
 
 public enum NodeType {
-    MASTER, WORKER, DEAD_SERVER
+    API, MASTER, WORKER, DEAD_SERVER
```

Comment:
It's better to put the new enum in the last index.

Reply:
ok